keel/api/v1/document/views.py

```diff
@@ -39,7 +42,14 @@ def generate(self, request, format = 'json', **kwargs):
         # doc_serializer = DocumentsSerializer(doc)
         # response_data = doc_serializer.data
         # response["data"] = response_data
-        file_handle = doc.avatar.open()
+        try:
+            file_handle = doc.avatar.open()
+        except FileNotFoundError as e:
```

Comment:
can we also log the exception here...
so that it can be tracked!

Reply:
@KeelTech I am yet to add it to any of the developed APIs. I will add the logs once for all the APIs and raise another PR. Do not want to keep this already dragged PR.

Reply:
I dont think its an effort..
lets not keep loose ends for now....
Still got many features ti wrap up  :) so lets closed with this in the best possible way we can!!